sdk/src/test/java/com/karhoo/sdk/api/service/payments/AdyenPaymentsDetailsInteractorTest.kt

```diff
@@ -0,0 +1,107 @@
+package com.karhoo.sdk.api.service.payments
+
+import com.karhoo.sdk.api.KarhooError
+import com.karhoo.sdk.api.network.response.Resource
+import com.karhoo.sdk.api.testrunner.base.BaseKarhooUserInteractorTest
+import com.nhaarman.mockitokotlin2.*
+import kotlinx.coroutines.CompletableDeferred
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.runBlocking
+import okhttp3.ResponseBody
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+
+class AdyenPaymentsDetailsInteractorTest : BaseKarhooUserInteractorTest() {
+
+    private val adyenPaymentsDetails: String = "{\"field1\":\"some text\", \"field2\":\"some more text\"}"
+    private val adyenPaymentsResponse: ResponseBody = mock()
+
+    private lateinit var interactor: AdyenPaymentsDetailsInteractor
+
+    @Before
+    override fun setUp() {
+        super.setUp()
+        whenever(credentialsManager.isValidToken).thenReturn(true)
+        interactor = AdyenPaymentsDetailsInteractor(credentialsManager, apiTemplate, context)
+    }
+
+    /**
+     * Given:   A request is made to get Adyen payments
+     * When:    The call is not successful
+     * Then:    An InternalSDKError is returned
+     **/
+    @Test
+    fun `get Adyen payments call failure not made if request is null`() {
```

Comment:
Minor comment but could you update so the comments and tests refer to Adyen payment details and not just Adyen payments

Reply:
Ah yes. Good shout